Product/Production/Adapters/General/CONNECTAdapterPDPOpenSSO/src/main/java/com/sun/identity/xacml/plugins/XSPAXACMLAuthzDecisionQueryHandler.java

```diff
@@ -194,10 +194,10 @@ public XSPAXACMLAuthzDecisionQueryHandler() {
             environmentLocality = getEnvironmentLocality(xacmlRequest);
 
             LOG.info("xspa.handleQuery():\n" + "userId = " + userId + "\n" + "Roles = " + userRoles + "\n"
-                + "resourceId = " + resourceId + "\n" + "purpose = " + pou + "\n" + "communityId = " + communityId
-                + "\n" + "serviceType = " + serviceType + "\n" + "OptIN = " + Boolean.toString(optIn) + "\n"
-                + "userLocality = " + userLocality + "\n" + "environmentLocality = " + environmentLocality + "\n"
-                + "actionId = " + actionId + "\n");
+                    + "resourceId = " + resourceId + "\n" + "purpose = " + pou + "\n" + "communityId = " + communityId
+                    + "\n" + "serviceType = " + serviceType + "\n" + "OptIN = " + Boolean.toString(optIn) + "\n"
```

Comment:
Can we user the new logging method here. Also there is no need to do Boolean.tostring(optIn), we caa simply concatenate the optIn variable.